src/Components/Steps/Steps.module.css

```diff
@@ -0,0 +1,149 @@
+.stepsContainer {
+  display: flex;
+  flex-direction: column;
+}
+
+.steppingContainer {
+  display: flex;
+  flex-direction: row;
+  cursor: pointer;
+  /* align-items: flex-start; */
+  /* justify-content: flex-start; */
+}
+.disbContainer {
+  position: relative;
+  margin: 10px -10px;
+  height: 45px;
+}
+
+.disbSteppingContainer {
+  position: absolute;
+  height: 50px;
+  left: 0;
+  background-color: var(--disabled-grey-background);
+  display: flex;
+  width: 200%;
+  border-radius: 5px;
+  font-weight: 500;
+  padding: 10px 0;
+}
+.stepNumber {
+  background-color: var(--purple);
+  width: 25px;
+  height: 25px;
+  margin-right: 7px;
+  text-align: center;
+  color: var(--white);
+  border-radius: 50%;
+  display: inline-block;
+}
+.stepText {
+  color: var(--black);
+  display: inline-block;
+}
+.disbStepNumber {
+  width: 25px;
+  height: 25px;
+  margin-right: 7px;
+  text-align: center;
+  color: var(--disabled-grey-text);
+  border: solid 1px var(--disabled-grey-text);
+  border-radius: 50%;
+  display: inline-block;
+  margin: 0 8px;
+}
+.disbStepText {
+  color: var(--disabled-grey-text);
+  display: inline-block;
+}
+
+.uploadBtnCont {
+  position: relative;
+  margin-bottom: 35px;
+  /* height: 45px; */
+  left: -20px;
+  border: 0;
+  background-color: white;
+}
+
+.uploadBtn {
+  position: absolute;
+  height: 45px;
+  left: 0;
+  background-color: var(--purple);
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  width: 310px;
+  border-radius: 5px;
+  font-weight: 500;
+  color: var(--white);
+  cursor: pointer;
+}
+
+.clickedUploadBtn {
+  position: absolute;
+  height: 45px;
+  left: 0;
+  background-color: var(--lightgrey);
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  width: 310px;
+  border-radius: 5px;
+  font-weight: 500;
+  color: var(--black);
+  cursor: pointer;
+}
+
+.continueBtn {
```

Comment:
`update`, `clicked`, and `continue` buttons seem to share a lot of properties. Consider breaking these into base and modifier classes, as described below in the `arrow` comment.